docs/build/references/command-line-interface.md

```diff
@@ -373,7 +373,7 @@ Enables the `web3_*` API. Defaults to true.
 
 `rpc-gas-cap` \(int\):
 
-The maximum gas to be consumed by an RPC Call \(used in `eth_estimateGas`\), measured in nAVAX \(GWei\). Defaults to 2,500,000,000.
+The maximum gas to be consumed by an RPC Call \(used in `eth_estimateGas`\), measured in nAVAX \(GWei\). Defaults to 50,000,000.
```

Comment:
This is not measured in nAVAX.

Reply:
@aaronbuchwald you can create a new PR or make changes in https://github.com/ava-labs/avalanche-docs/pull/419